### DISCUSSION TITLE
Master System Games with SRAM

### DISCUSSION THREAD

**Discussion by Mythris29** [sanni/cartreader#473] [Ideas & Suggestions]
Just wondering what it would take to get support for Master System games with batteries (SRAM). I'll provide some pics from the game I have and I'm happy to provide more info as needed. I'm not familiar with coding but I'd love to support this in other ways if somebody could take this on.

![PXL_20220816_180220909](https://user-images.githubusercontent.com/15204330/184949200-5d9859a8-bb1e-424d-9c46-4e9b2de40e88.jpg)
![PXL_20220816_180228019](https://user-images.githubusercontent.com/15204330/184949204-52bc9ef2-7b22-4bd1-9f19-101c7cae8112.jpg)
![PXL_20220816_180307987](https://user-images.githubusercontent.com/15204330/184949207-d5a55ee4-cc3b-4d1a-83fa-0234a0c371f5.jpg)
![PXL_20220816_180342028](https://user-images.githubusercontent.com/15204330/184949208-68e0304a-2c89-4619-a22b-fdf94cfec65a.jpg)
![PXL_20220816_180534037](https://user-images.githubusercontent.com/15204330/184949210-34959378-436f-45b1-ab0a-f1f7b544f87f.jpg)
![PXL_20220816_180602793](https://user-images.githubusercontent.com/15204330/184949213-9b13862e-decd-473e-9413-c26eb7465337.jpg)
![PXL_20220816_180644356](https://user-images.githubusercontent.com/15204330/184949215-7447dbbb-4b3f-4c12-ae8b-a2d3b33ee770.jpg)
.

**Comment by Mythris29** [comment]
Last pic added just to show that the slot works :)

**Reply by PsyK0p4T** [reply]
you probably did already but just in case, have you tried to brush hard (and many times) the cartridge pins ?
With q-tip, isopropyl alcohol... and retry the dump process.

**Reply by Mythris29** [reply]
Always good to check :) but yes I did already do that. It comes up with a different checksum everytime so I don't think it's being read correctly. I believe I read something that this type of master system cartridge isn't supported yet so I'm assuming that's the issue.
